injector/grpc_disruption.go

```diff
@@ -98,10 +102,11 @@ func (i *GRPCDisruptionInjector) Clean() error {
 	return conn.Close()
 }
 
-func connectToServer(serverAddr string) (*grpc.ClientConn, error) {
+func connectToServer(serverAddr string, timeout time.Duration) (*grpc.ClientConn, error) {
```

Comment:
As this seems part of our struct responsibility, I may move this method to a struct related, hence no more need of providing arguments at all.